content/ios/record.md

```diff
@@ -153,3 +154,333 @@ method.
 
 You can also delete multiple records at once.
 (TODO: Add code example)
+
+<a name="auto-increment"></a>
+# Auto-Incrementing Sequence Fields
+
+Skygear reserves the `id` field in the top level of all record as a primary key.
+`id` must be unique and default to be Version 4 UUID. If you want to
+auto-incrementing id for display purpose, Skygear provide a sequence for this 
+purpose. The sequence is guaranteed unique.
```

Comment:
For this paragraph with the few notes below the code, it will be better for you to discuss with @TakLee96 so the JS guide and iOS guide will be consistent. If it's decided that the JS guide has to be changed, you can change that in the same commit as you modify the iOS guide as well